A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Elec.js

```diff
@@ -0,0 +1,1081 @@
+                    this.setCONNECTION_AC2_ACESS_ON()
+                    this.setCONNECTION_AC1_ACESS_OFF()
+                    this.buses.ACESS = 'ON'
+                }
+            }
+            if (this.buses.GALLEY_SHED === true) {
+                this.setGALLEY_SHED_ON()
+            } else {
+                this.setGALLEY_SHED_OFF()
+            }
+    
+            // Level 3 - TR
+            if (this.systems.TR1 === 'ON') {
+                this.setCONNECTION_TR1_DC1_ON()
+                this.buses.DC1 = 'ON'
+            }
+            if (this.systems.TR2 === 'ON') {
+                this.setCONNECTION_TR2_DC2_ON()
+                this.buses.DC2 = 'ON'
+            }
+    
+            // Level 4A - Batteries
+            // if (this.systems.BAT1 === 'ON') {
+            //     this.setBAT1_ON()
+            //     if (this.internal.BAT1_DIR === 'CHARGE') {
+            //         this.setCONNECTION_DCBAT_BAT1_OFF()
+            //         this.setCONNECTION_BAT1_DCBAT_DISCHARGE_OFF()
+            //         this.setCONNECTION_DCBAT_BAT1_CHARGE_ON()
+            //     } else if (this.internal.BAT1_DIR === 'DISCHARGE') {
+            //         this.setCONNECTION_DCBAT_BAT1_OFF()
+            //         this.setCONNECTION_DCBAT_BAT1_CHARGE_OFF()
+            //         this.setCONNECTION_BAT1_DCBAT_DISCHARGE_ON()
+            //     } else if (this.internal.BAT1_DIR === 'IDLE') {
+            //         this.setCONNECTION_DCBAT_BAT1_CHARGE_OFF()
+            //         this.setCONNECTION_BAT1_DCBAT_DISCHARGE_OFF()
+            //         this.setCONNECTION_DCBAT_BAT1_ON()
+            //     } else {
+            //         this.setCONNECTION_DCBAT_BAT1_OFF()
+            //         this.setCONNECTION_DCBAT_BAT1_CHARGE_OFF()
+            //         this.setCONNECTION_BAT1_DCBAT_DISCHARGE_OFF()
+            //     }
+            // } else {
+            //     this.setBAT1_OFF()
+            //     if (this.internal.BAT2_DIR === 'CHARGE') {
+            //         this.setCONNECTION_DCBAT_BAT2_OFF()
+            //         this.setCONNECTION_BAT2_DCBAT_DISCHARGE_OFF()
+            //         this.setCONNECTION_DCBAT_BAT2_CHARGE_ON()
+            //     } else if (this.internal.BAT2_DIR === 'DISCHARGE') {
+            //         this.setCONNECTION_DCBAT_BAT2_OFF()
+            //         this.setCONNECTION_DCBAT_BAT2_CHARGE_OFF()
+            //         this.setCONNECTION_DCBAT_BAT2_DISCHARGE_ON()
+            //     } else if (this.internal.BAT2_DIR === 'IDLE') {
+            //         this.setCONNECTION_DCBAT_BAT2_CHARGE_OFF()
+            //         this.setCONNECTION_BAT2_DCBAT_DISCHARGE_OFF()
+            //         this.setCONNECTION_DCBAT_BAT2_ON()
+            //     } else {
+            //         this.setCONNECTION_DCBAT_BAT2_OFF()
+            //         this.setCONNECTION_DCBAT_BAT2_CHARGE_OFF()
+            //         this.setCONNECTION_BAT2_DCBAT_DISCHARGE_OFF()
+            //     }
+            // }
```

Comment:
Is this commented out because it doesn't work ? Will it be added when this PR is ready ?